### PR TITLE
Added test cases for the case review apis.

### PR DESCRIPTION
**Pull request by ShifaSZ**
Added test cases for the case review apis.

### CODE REVIEW

File: seqr/views/apis/case_review_api_tests.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        }
+        response = self.client.post(url, content_type='application/json',
+                                    data=json.dumps(req_values))
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+
+        self.assertListEqual(response_json.keys(), [FAMILY_GUID])
+        self.assertEqual(response_json[FAMILY_GUID]['familyGuid'], FAMILY_GUID)
+        self.assertEqual(response_json[FAMILY_GUID]['internalCaseReviewSummary'], req_values['value'])
+        # send request with a "value" attribute
```
Comment (hanars) [inline seqr/views/apis/case_review_api_tests.py]: I think you meant this to say "send request with an INVALID "value" attribute"

File: seqr/views/apis/case_review_api_tests.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+import json
+
+from django.core.files.uploadedfile import SimpleUploadedFile
```
Comment (hanars) [inline seqr/views/apis/case_review_api_tests.py]: We have an automated code quality checker called codacy. The integration with github is not working, but it flagged this line as an unused import: https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5244461

Comment (hanars) [inline seqr/views/apis/case_review_api_tests.py]: We have an automated code quality checker called codacy. The integration with github is not working, but it flagged this line as an unused import: https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5244461

File: seqr/views/apis/case_review_api_tests.py (hunk 1 above)
Comment (ShifaSZ) [reply seqr/views/apis/case_review_api_tests.py]: Fixed it.

File: seqr/views/apis/case_review_api_tests.py (hunk 2 above)
Comment (ShifaSZ) [reply seqr/views/apis/case_review_api_tests.py]: Deleted.

**Pull request merged**